_posts/2017-10-25-myget-by-the-numbers-2017.markdown

```diff
@@ -0,0 +1,76 @@
+
+As MyGet has been growing, we've also been facing various challenges in terms of scalability. In the face of those challenges, we're proud on our uptime numbers and have been working hard on improving MyGet's scalability and reliability. We still have work to do to further improve performance which will receive great attention going forward. We figured it'd be an interesting story to share with you. Of course we can’t share all of our statistics, but it’s always fun to look at them. 
+
+## Shared Infrastructure
+
+MyGet is hosted on Microsoft Azure and has regional deployments in Europe and US. MyGet has a shared tenant, [www.myget.org](https://www.myget.org), home of the personal subscription plans (Free, Starter and Professional subscriptions). [MyGet Enterprise](https://www.myget.org/enterprise) tenants run on their own dedicated infrastructure, which may differ from what is outlined below.
+
+The shared tenant has its primary deployment in the West Europe region, and a secondary in US East. We use Azure Cloud Services to deploy both Web and Worker roles, and apply auto-scaling policies on both to ensure the service scales as needed.
+
+On average, the shared tenant has 4 Web roles and 4 worker roles deployed in each region, and scales in and out multiple times (minimum 2 instances per role, up to 8 during peak moments). Judging by the observed peak moments, all of us restore most packages in the morning (which is multiple times a day when running a global service).
+
+## Reliability
+
+Being developers ourselves, we know how frustrating it is when a service you use on a daily basis goes down. Even though we cannot offer an official SLA just yet, we can demonstrate historical uptime data on our [status page](https://www.myget.org/status). We invite you to go take a look if interested (and it lists uptime data for other endpoints too). 
+
+The most interesting data point for package consumers is likely the following one: package download uptime. We're happy to share that, in 2017, we're currently on track to exceed the 99.99% uptime goal we set ourselves!
+
+Currently, the package download endpoint has an uptime percentage of **99.992%**! In other words, the endpoint has been down for 35 minutes in 2017, spread out over 5 different days, so that's 5 days this year with an average interruption of less than 7 minutes. If you didn't do a restore during those brief interruptions, you likely haven't noticed this interruption at all. If you did, we're very sorry, and please know that we'll continue to work hard to further improve on this metric.
```

Comment:
Number maybe needs revision?

Reply:
Yes, we had 5mins outage last night (EU West package downloading endpoint was little affected)